src/schema/basic_slots.yaml

```diff
@@ -603,6 +603,12 @@ enums:
 
   FileTypeEnum:
     permissible_values:
+      
+      Reference Calibration File:
+        description: A file that contains data used to calibrate a computational software tool.
+        comments:
+          - A general term meant to capture reference calibration files broadly.
```

Comment:
@turbomam does this work?

Reply:
Can I ponder (or can we talk it though) over the course of today?

I can think of several kinds of reference calibration files but am trying to decide if they are worth exploring.

I guess we can focus on this question: if a user searched for data objects whose file type was "Reference Calibration File" would they get everything they want and only what they want? Or is it likely that they would have to filter on other aspects of the data object, or on things that have a relationship with the data object?

Reply:
They would probably have to filter on other aspects of the data object, like `name` and `description` of the `DataObject` in addition to the `FileType` would clarify what kind of calibration file. I don't think that is necessarily a bad thing. Much of the time more advanced queries that require more than one attribute are needed for a user to narrow down their searches. I think because all of that information would be in one class instance - that should be okay. If they had to traverse the schema to figure out more information about the calibration file - that might be problematic as you are describing since that is no easy task.

Reply:
My reasoning behind leaving it more general is that there will only be a handful of these data objects. For example, NOM will reuse 2 or three of these calibration files. We may add a few more in the future. But its a small amount. So if we want to get more specific in the `FileTypeEnum` value for calibration files we can. But leaving it more general will still only return a small handful of results. A very reasonable amount for a user to look through.

Reply:
OK, that is very helpful feedback.

I approve and will add a note about this to 
- https://github.com/microbiomedata/nmdc-schema/issues/2186

Reply:
Would it be incorrect to say that all of the current use cases involve either a chromatographic separation or a mass spectrometer? If so, I suggest adding that to the commentary. But that doesn't change my previous approval.

Reply:
Yes, I think so far that is all the use cases. Specicially for metabolomics and nom analyses. Unless @aclum can speak about calibration files for metagenomics or metatranscriptomics?

Reply:
@turbomam I changed the description to be specific to Nom and Metabolomics analysis. Would you like me to remove the comment since it clashes with the specificity of the description?

Reply:
yes, in my opinion that is a very clear description, so the conflicting comment should be removed. Thanks for your follow-though.